wiktextract/data/de/form_tables.json

```diff
@@ -0,0 +1,119 @@
+[
+  "Deutsch Substantiv Übersicht",
```

Comment:
I'd prefer not including lots of hard coded data at the beginning, I think this long template list is unnecessary for these reasons:
- most can be found by a simple `template_name.endswith("Übersicht")` 
- and all these table templates won't be pre-expanded thus won't needed to be included in the `do_not_pre_expand` parameter. If they are pre-expaned then the problem is `Wtp.analyze_templates()` function marks too many pre-expand templates and should only be called by the English Wiktionary code and in the future we should make the pre-expand rules configurable like only check if the template contains list and ignore other cases.

I'd recommend you also take a look of the newer French extractor code which is simpler than the English extractor and doesn't use much hard code data and regex.